### PR TITLE
{chem}[iomkl/2017a] NWChem v6.6.revision27746

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
NWchem 6.6 using iomkl/2017a and Python 2.7.12. Depends on new nwchem easyblock from https://github.com/hpcugent/easybuild-easyblocks/pull/1104

### CODE REVIEW

File: easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-iomkl-2017a-2015-10-20-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'iomkl', 'version': '2017a'}
+toolchainopts = {'i8': True}
+
+source_urls = ['http://www.nwchem-sw.org/download.php?f=']
+verdate = '2015-10-20'
+sources = ['Nwchem-%s-src.%s.tar.bz2' % (version, verdate)]
+
+patches = [
+    'NWChem_fix-date.patch',
+    'NWChem-%(version)s-parallelbuild.patch',
```
Comment (boegel) [inline easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-iomkl-2017a-2015-10-20-Python-2.7.12.eb]: @akesandgren missing patch file?

Comment (boegel) [inline easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-iomkl-2017a-2015-10-20-Python-2.7.12.eb]: :+1:

File: easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-parallelbuild.patch [hunk 2]
```diff
@@ -0,0 +1,80 @@
+diff -ru nwchem-6.6.orig/src/config/makefile.h nwchem-6.6/src/config/makefile.h
```
Comment (boegel) [inline easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-parallelbuild.patch]: @akesandgren can you include a brief description of this patch at the top, explaining what it does, where it comes from and/or who the author is?

Comment (akesandgren) [inline easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-parallelbuild.patch]: Done. It was taken from the existing NWchem 6.5 of course but re-done with diff to match correctly and i forgot to re-add the comment at the top :-(

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2138.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/347fd458c232a9cb4b577785777af233 for a full test report.

**Comment by boegel** [comment]
@akesandgren do ignore the failing test reports, I'm running on too many NWChem builds in parallel :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2521.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ded7922a08a8d0a6523cd5dd858e39ba for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2154.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8479de8e89a9d103261604cc4187a6fc for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**